test/e2e/plain_provisioner_test.go

```diff
@@ -45,7 +45,12 @@ var _ = Describe("plain provisioner bundle", func() {
 				},
 				Spec: rukpakv1alpha1.BundleSpec{
 					ProvisionerClassName: plainProvisionerID,
-					Image:                "quay.io/tflannag/olm-plain-bundle:olm-crds-v0.20.0",
+					Source: rukpakv1alpha1.BundleSource{
+						Type: "image",
```

Comment:
Would you add constants for types in the bundle_types.go ?

Reply:
Yep - good catch.